plugin/completion/lib_complete.py

```diff
@@ -143,7 +147,8 @@ def parse_tu(self, view):
                     args=self.clang_flags,
                     unsaved_files=files,
                     options=TU.PARSE_PRECOMPILED_PREAMBLE |
-                    TU.PARSE_CACHE_COMPLETION_RESULTS)
+                    TU.PARSE_CACHE_COMPLETION_RESULTS |
+                    TU.PARSE_INCLUDE_BRIEF_COMMENTS_IN_CODE_COMPLETION)
```

Comment:
@simia I do sleep, quite a lot actually :laughing: Just this night went to bed really early, so was up at like 6 am. I promise, this won't happen too often :)

I comment the code like this:
```
/**
 * @brief      blah blah blah.
 */
```

I will try with your suggestion. It's not a problem to change that style, as soon as it can be done automatically :) I definitely don't want to parse all the variety of possible Doxygen styles available.